src/enums.ts

```diff
@@ -0,0 +1,161 @@
+import { z } from 'zod'
```

Comment:
Instead of a catch all enum file, wonder if we can just define enums in the corresponding places where it's used. E.g., kyc enums are in the kyc.ts file, transfer enums are in the transfer.ts file, etc.
Although not sure if we'll end up with complex dependencies since webhook stuff may go in a separate file, but will need the kyc / transfer enums

Reply:
Yep i can do that, it looks like most of the types split pretty well into a specific endpoint.